app/views/achievements/_grid.html.haml

```diff
@@ -1,11 +1,13 @@
 - player ||= nil
 .row.achievement_grid
   - achievements.each do |achievement|
-    .col-md-3.achievement
+    .col-md-4.achievement
       .panel.panel-default{class: achievement_class(achievement, player)}
-        .panel-heading=achievement.title
+        .panel-heading
+          = achievement.title
+          .points.pull-right{title: t("achievement_points")}
```

Comment:
> Nitpick

And it's totes not nitpicking. The `desperado` codebase is very consistent and as such it is a pleasure to crawl over it. Good stuff Johno, have yourself a double mocha latte decaf two shots, you're worth it.

:coffee: :smiley_cat: